ddtrace/bootstrap/sitecustomize.py

```diff
@@ -46,6 +48,8 @@ def update_patched_modules():
     patch = True
 
     # Respect DATADOG_* environment variables in global tracer configuration
+    # TODO: these variables are deprecated; use utils method and update our documentation
```

Comment:
Added a TODO. Not related to this PR but something we should tackle sooner or later.